### ISSUE TITLE
Create a new get_page_title() function

### ISSUE THREAD

**Issue by bbertucc** [bbertucc/equalify#273]
## Overview
Solving the problem on screenreaders:
> In the list of four header nav items, Reports and Scans are always announced as current regardless of where you actually are. For example, after navigating to My Account, the two items are still announced as such, even if My Account is the current page.

We'll need to create a new helper function that can be loaded into `index.php`. The helper function can be modeled after existing helpers, like [get_content()](https://github.com/EqualifyEverything/v1/blob/main/helpers/get_content.php).

That pull in the page title can be built using the view and the id in the URL parameter. Note: ID parameters are sometime empty, so that should be optional. If nothing exists, it should default to the current title: `Equalify | Accessibility Issue Management`. 

## Additional Information
We will need to make sure Property Settings, Report, Report Settings, Message, Tag, and Page titles also reference the report. You can get those titles via a db lookup. 

Here is the general semantic of each view
- No view specified: Equalify | Accessibility Management
- **property_settings**: `[property name (if exists)]` Property Settings | Equalify
- **reports**: All Reports | Equalify
- **scans**: Scans | Equalify
- **settings**: Settings | Equalify
- **account**: My Account | Equalify
- **report**: `[report name (if exists)]` Report | Equalify
- **message**: `[message_title (if exists), truncated at 25 characters]` Message Detail | `[report name (if exists)]` Report | Equalify
- **tag**: `[tag_name (if exists), truncated at 25 characters]` Tag Detail | `[report name (if exists)]` Report | Equalify
- **page**: `[page_url (if exists), truncated at 25 characters]` Page Detail | `[report name (if exists)]` Report | Equalify

For instance:
- `index.php?view=property_settings&property_id=15` has the page title "BBPress.org Property Settings | Equalify"
- `index.php?view=report&report_id=20` has the page title "WordPress Properties Report | Equalify"
- `index.php?view=message&report_id=20&message_id=61` has the page title "Element has insufficient.. Message Detail | WordPress Properties Report | Equalify"
- `view=page&report_id=20&page_id=4061` has the page title "https://wordpress.org/stat.. Page Detail | Equalify"
- `index.php?view=tag&report_id=20&tag_id=2` has the page title "wcag2aa Tag Detail | WordPress Properties Report | Equalify"

More info on titles: https://www.w3.org/WAI/WCAG21/Understanding/page-titled.html

**Comment by jwgoedert** [comment]
It seems there are two issues at play:
1. Creating a title dynamically
2. Setting the current page in Navigation
 Index.php has reports and scans hardcoded as current with aria-current="page" attribute. 

Pushed up fixes-Id's only captured on "reports" are there IDs or other info necessary on scans/settings/account?

**Comment by bbertucc** [comment]
In addition to reports, we'll need to specify page names on message, page, and tag detail pages.

**Comment by jwgoedert** [comment]
Good deal, there's currently a switch case handling different page names, I'll go off the assumption that the "view" values will be "message", "page" and "tag_detail" and update

**Comment by bbertucc** [comment]
> Good deal, there's currently a switch case handling different page names, I'll go off the assumption that the "view" values will be "message", "page" and "tag_detail" and update

that sounds like the right assumption

**Comment by bbertucc** [comment]
@jwgoedert - I added some specifications to the original issue that should clarify what we're after.
